### PR TITLE
add smp_utils version 0.98

### PR DESCRIPTION
**Pull request by grueni**
Please check the handling of the license. I copied the type of the license from smp_utils.spec and the content from COPYING.

### CODE REVIEW

File: components/sysutils/smp_utils/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+PKG_PROTO_DIRS += $(COMPONENT_DIR)/files
+
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+test:	 $(TEST_32)	
+
+REQUIRED_PACKAGES += system/library
```
Comment (xen0l) [inline components/sysutils/smp_utils/Makefile:47]: did you run gmake REQUIRED_PACKAGES, or you added this just by hand?

Comment (grueni) [reply components/sysutils/smp_utils/Makefile:47]: I run make REQUIRED_PACKAGES.

File: components/sysutils/smp_utils/Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+CFLAGS += -I$(COMPONENT_DIR)/$(COMPONENT_SRC)/include
+
+CONFIGURE_OPTIONS += CFLAGS="$(CFLAGS)"
+CONFIGURE_OPTIONS += --bindir ${CONFIGURE_SBINDIR.32}
+
+# Move the section 8 man pages into section 1m.
+COMPONENT_POST_INSTALL_ACTION = \
```
Comment (pyhalov) [inline components/sysutils/smp_utils/Makefile]: This looks ugly. If component is automake/autoconf-based, it can be rather simple to patch Makefile.am

Comment (alarcher) [reply components/sysutils/smp_utils/Makefile]: Or just use a transform.

Comment (grueni) [reply components/sysutils/smp_utils/Makefile]: This is a copy of the same proceeding in sg3_utils. Same author, same attitude.
configure has no options to define the target for the man files.
In folder doc I find the files with extension ".8" and the Makefile.am using this files.
Shall I rename *.8 to *.1 and change the content of Makefile.am?
I fear this won't be look nicer.

Comment (grueni) [reply components/sysutils/smp_utils/Makefile]: transform sounds interesting, will try it, give me some time

**Comment by grueni** [comment]
Squash the 2 commits in one?

**Pull request merged**